tests/test_command_handler.py

```diff
@@ -101,13 +101,15 @@ async def test_get_telemetrey(self):
         self.assertEqual(self.response["response"], ResponseCode.OK)
         self.assertTrue(self.command_handler._started)
 
+        # Give some time to the mock sensor to produce data
         await asyncio.sleep(2)
+        telemetry = self.response["telemetry"]
         # TODO: Properly support multiple sensors (DM-30070)
         config = self.configuration["devices"][0]
         for i in range(3, config["channels"] + 3):
             self.assertTrue(
                 MockTemperatureSensor.MIN_TEMP
-                <= self.response[i]
+                <= telemetry[i]
```

Comment:
You'll probably get a better error message (with pytest, at least) if you use:
```
assert MockTemperatureSensor.MIN_TEMP <= telemetry[i] <= ...
```
The problem with assertTrue is that if it fails, the message is pretty useless.

Reply:
Thanks, I wasn't aware of that construct. Done.